### PR TITLE
[Merged by Bors] - fix: use modern ubuntu version for gitpod

### PR DESCRIPTION
**Pull request by gebner**
This fixes `lake exe cache get` in gitpod.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
We might need to copy some stuff from https://github.com/gitpod-io/workspace-images/blob/main/base/Dockerfile if we update the image.

File: .docker/gitpod/Dockerfile [hunk 1]
```diff
@@ -2,19 +2,21 @@
 
 # gitpod doesn't support multiple FROM statements, (or rather, you can't copy from one to another)
 # so we just install everything in one go
-FROM gitpod/workspace-base
+FROM ubuntu:jammy
 
 USER root
 
-RUN apt-get update && apt-get install curl git python3 python3-pip python3-venv -y
+RUN apt-get update && apt-get install git curl git -y && apt-get clean
```
Comment (eric-wieser) [inline .docker/gitpod/Dockerfile:9]: https://www.gitpod.io/docs/configure/workspaces/workspace-image#custom-base-image suggests we should install `sudo` too.

**Comment by gebner** [comment]
> It's probably worth installing the latest git version, since the default image does

I don't think that's enough of a reason.  I would only add the extra PPA if you want a git feature from git > 2.34.  (Upgrading git makes a lot more sense for ancient ubuntu versions which have a correspondingly ancient git version.)

**Comment by gebner** [comment]
I've added all the other goodies you've pointed out.

**Comment by gebner** [comment]
bors merge

**Pull request closed without merging**